### ISSUE TITLE
WebVTT captions with voice spans will cause errors on version 6.2.4

### ISSUE THREAD

**Issue by DoomTay** [videojs/video.js#4542]
## Description

[Test case](http://jsbin.com/ceqaradeco/1/edit?html,console,output)

(JS Bin's cross-bin referencing seems to not work for subtitle files, so I used a Data URL)

## Steps to reproduce
1. Open the above link
1. Play the video

### Expected
Four lines of dialogue should appear throughout the first 10 seconds of the video

### Actual
No subtitles appear, instead each time a line i set to come up, an error message appears instead

#### Firefox console
```
VIDEOJS: ERROR: TypeError: setting a property that has only a getter
Stack trace:
createElement@http://vjs.zencdn.net/6.2.4/video.js:8321:5
parseContent@http://vjs.zencdn.net/6.2.4/video.js:8360:14
CueStyleBox@http://vjs.zencdn.net/6.2.4/video.js:8538:17
WebVTT$1.processCues/<@http://vjs.zencdn.net/6.2.4/video.js:8997:18
WebVTT$1.processCues@http://vjs.zencdn.net/6.2.4/video.js:8990:4
updateForTrack@http://vjs.zencdn.net/6.2.4/video.js:12050:5
updateDisplay@http://vjs.zencdn.net/6.2.4/video.js:12021:7
bound@http://vjs.zencdn.net/6.2.4/video.js:2232:12
bound@http://vjs.zencdn.net/6.2.4/video.js:2232:12
on/data.dispatcher@http://vjs.zencdn.net/6.2.4/video.js:1867:15
trigger@http://vjs.zencdn.net/6.2.4/video.js:1994:5
trigger$$1@http://vjs.zencdn.net/6.2.4/video.js:2795:12
@http://vjs.zencdn.net/6.2.4/video.js:23357:12
bound@http://vjs.zencdn.net/6.2.4/video.js:2232:12
bound@http://vjs.zencdn.net/6.2.4/video.js:2232:12
on/data.dispatcher@http://vjs.zencdn.net/6.2.4/video.js:1867:15
trigger@http://vjs.zencdn.net/6.2.4/video.js:1994:5
trigger$$1@http://vjs.zencdn.net/6.2.4/video.js:2795:12
updateDisplay@http://vjs.zencdn.net/6.2.4/video.js:10445:14
on/data.dispatcher@http://vjs.zencdn.net/6.2.4/video.js:1867:15
trigger@http://vjs.zencdn.net/6.2.4/video.js:1994:5
EventTarget.prototype.trigger@http://vjs.zencdn.net/6.2.4/video.js:2419:3
TextTrack/timeupdateHandler<@http://vjs.zencdn.net/6.2.4/video.js:7378:9
bound@http://vjs.zencdn.net/6.2.4/video.js:2232:12
bound@http://vjs.zencdn.net/6.2.4/video.js:2232:12
on/data.dispatcher@http://vjs.zencdn.net/6.2.4/video.js:1867:15
  video.js:489:5
```

#### Chrome console
```
VIDEOJS: ERROR: TypeError: Cannot assign to read only property 'localName' of object '#<HTMLSpanElement>'
    at createElement (video.js:8321)
    at parseContent (video.js:8360)
    at new CueStyleBox (video.js:8538)
    at video.js:8997
    at Function.WebVTT$1.processCues (video.js:9009)
    at TextTrackDisplay.updateForTrack (video.js:12050)
    at TextTrackDisplay.updateDisplay (video.js:12021)
    at Player.bound (video.js:2232)
    at HTMLDivElement.bound (video.js:2232)
    at HTMLDivElement.data.dispatcher (video.js:1867)
logByType @ video.js:489
```

#### videojs
6.2.4
#### browsers
Firefox 54.0.1, Chrome  60.0.3112.90
#### OSes
Windows 10
### plugins
None

**Comment by gkatsev** [comment]
That's weird. It looks like the issue only happens when vtt.js is included in with video.js and works fine if you use the novtt build of video.js. See https://output.jsbin.com/kinonoj.

**Comment by gkatsev** [comment]
After some frustration, I realized that this is because vttjs is now being included in strict mode where-as previously it wasnt and when using the novtt build it also isn't included in strict mode.

**Comment by gkatsev** [comment]
Should be fixed in 6.2.6.
